components/object-property-list/test/object-property-list-item.test.js

```diff
@@ -0,0 +1,17 @@
+import '../object-property-list-item.js';
+import { expect, fixture, html } from '@open-wc/testing';
+
+const getIcon = item => item.shadowRoot.querySelector('.item-icon');
+
+describe('d2l-object-property-list-item', () => {
+
+	it('should show/hide the icon when its property is changed', async() => {
+		const elem = await fixture(html`<d2l-object-property-list-item text="Example item"></d2l-object-property-list-item>`);
+		expect(getIcon(elem)).to.be.null;
+
+		elem.icon = 'emoji:happy';
+		await elem.updateComplete;
+		expect(getIcon(elem)).not.to.be.null;
```

Comment:
Is this really testing much beyond that Lit is correctly binding the icon property? I'm not sure either of these item tests is really adding much value... might be better to just stick with the list one.